### PR TITLE
more span macro changes in brain traumas and disease files

### PR DESCRIPTION
**Pull request by flowercuco**
## About The Pull Request

i was fucking around with brain traumas on a downstream and noticed they had similar issues to quirks so i decided to continue work from #73116 

Code_Klx14O288V
(search in VSC for span class = 'notice')
its going to be a bit of a thing to get all of these but this is a decent chunk i think

there was only one annoying/tough file.
imaginary_friend.dm had class = 'game say' and class = 'emote' both of which after some testing did not seem like they did anything. ill try to keep that in mind in other files if i continue to do this but i either omitted them because they didnt have any formatting or, in the case of emote, turned it into name, which i think is what you'd want those messages to look like.

there were also a few small spelling errors that i fixed

## Why It's Good For The Game

more consistent and stops people from copying brain trauma formatting wrong

## Changelog

they should all work the same

### CODE REVIEW

File: code/datums/brain_damage/hypnosis.dm [hunk 1]
```diff
@@ -25,14 +25,14 @@
...
-	to_chat(owner, "<span class='notice'>[pick("You feel your thoughts focusing on this phrase... you can't seem to get it out of your head.",\
+	to_chat(owner, span_reallybig(span_hypnophrase("[hypnotic_phrase]")))
+	to_chat(owner, span_notice("[pick("You feel your thoughts focusing on this phrase... you can't seem to get it out of your head.",\
 												"Your head hurts, but this is all you can think of. It must be vitally important.",\
 												"You feel a part of your mind repeating this over and over. You need to follow these words.",\
 												"Something about this sounds... right, for some reason. You feel like you should follow these words.",\
-												"These words keep echoing in your mind. You find yourself completely fascinated by them.")]</span>")
-	to_chat(owner, "<span class='boldwarning'>You've been hypnotized by this sentence. You must follow these words. If it isn't a clear order, you can freely interpret how to do so,\
-										as long as you act like the words are your highest priority.</span>")
+												"These words keep echoing in your mind. You find yourself completely fascinated by them.")]"))
```
Comment (san7890) [inline code/datums/brain_damage/hypnosis.dm:33]: make this a picklist while you're in the area so we don't have to do these ugly slashes and horrid indentation (artifact from when you couldn't multiline lists):

```dm
to_chat(owner, span_notice("[pick(list(
	"You feel your thoughts focusing on this phrase... you can't seem to get it out of your head.",
	"Your head hurts, but this is all you can think of. It must be vitally important.",
	"You feel a part of your mind repeating this over and over. You need to follow these words.",
	"Something about this sounds... right, for some reason. You feel like you should follow these words.",
	"These words keep echoing in your mind. You find yourself completely fascinated by them.",
))]"))
```

also alphabetize it

File: code/datums/brain_damage/hypnosis.dm [hunk 2]
```diff
@@ -25,14 +25,14 @@
...
+	to_chat(owner, span_boldwarning("You've been hypnotized by this sentence. You must follow these words. If it isn't a clear order, you can freely interpret how to do so, \
+										as long as you act like the words are your highest priority."))
```
Comment (san7890) [inline code/datums/brain_damage/hypnosis.dm:35]: I think you can change this to fit all one line? might be long though.

File: code/datums/brain_damage/special.dm [hunk 3]
```diff
@@ -324,13 +324,13 @@
...
 	veil = new(owner.drop_location())
-	to_chat(owner, "<span class='warning'>[pick("You stop thinking for a moment. Therefore you are not.",\
+	to_chat(owner, span_warning("[pick("You stop thinking for a moment. Therefore you are not.",\
 												"To be or not to be...",\
 												"Why exist?",\
 												"You stop keeping it real.",\
 												"Your grip on existence slips.",\
 												"Do you even exist?",\
-												"You simply fade away.")]</span>")
+												"You simply fade away.")]"))
```
Comment (san7890) [inline code/datums/brain_damage/special.dm:333]: Make this a picklist with the same formatting as the other suggestion as well.

File: code/datums/diseases/transformation.dm [hunk 4]
```diff
@@ -119,9 +119,9 @@
 	stage1 = list()
 	stage2 = list()
 	stage3 = list()
-	stage4 = list("<span class='warning'>Your back hurts.</span>", "<span class='warning'>You breathe through your mouth.</span>",
-					"<span class='warning'>You have a craving for bananas.</span>", "<span class='warning'>Your mind feels clouded.</span>")
-	stage5 = list("<span class='warning'>You feel like monkeying around.</span>")
+	stage4 = list(span_warning("Your back hurts."), span_warning("You breathe through your mouth."),
+					span_warning("You have a craving for bananas."), span_warning("Your mind feels clouded."))
```
Comment (san7890) [inline code/datums/diseases/transformation.dm:123]:
```suggestion
	stage4 = list(
		span_warning("Your back hurts."),
		span_warning("You breathe through your mouth."),
		span_warning("You have a craving for bananas."),
		span_warning("Your mind feels clouded.",
	))
```

Multi-line this list like this.

File: code/datums/diseases/transformation.dm [hunk 5]
```diff
@@ -154,10 +154,10 @@
 	severity = DISEASE_SEVERITY_BIOHAZARD
 	visibility_flags = NONE
 	stage1 = list()
-	stage2 = list("Your joints feel stiff.", "<span class='danger'>Beep...boop..</span>")
-	stage3 = list("<span class='danger'>Your joints feel very stiff.</span>", "Your skin feels loose.", "<span class='danger'>You can feel something move...inside.</span>")
-	stage4 = list("<span class='danger'>Your skin feels very loose.</span>", "<span class='danger'>You can feel... something...inside you.</span>")
-	stage5 = list("<span class='danger'>Your skin feels as if it's about to burst off!</span>")
+	stage2 = list("Your joints feel stiff.", span_danger("Beep...boop.."))
+	stage3 = list(span_danger("Your joints feel very stiff."), span_warning("Your skin feels loose."), span_danger("You can feel something move...inside."))
```
Comment (san7890) [inline code/datums/diseases/transformation.dm:158]: Optional to multiline this list, but would be nice.

File: code/datums/diseases/transformation.dm [hunk 6]
```diff
@@ -191,10 +191,10 @@
 	severity = DISEASE_SEVERITY_BIOHAZARD
 	visibility_flags = NONE
 	stage1 = list()
-	stage2 = list("Your throat feels scratchy.", "<span class='danger'>Kill...</span>")
-	stage3 = list("<span class='danger'>Your throat feels very scratchy.</span>", "Your skin feels tight.", "<span class='danger'>You can feel something move...inside.</span>")
-	stage4 = list("<span class='danger'>Your skin feels very tight.</span>", "<span class='danger'>Your blood boils!</span>", "<span class='danger'>You can feel... something...inside you.</span>")
-	stage5 = list("<span class='danger'>Your skin feels as if it's about to burst off!</span>")
+	stage2 = list("Your throat feels scratchy.", span_danger("Kill..."))
+	stage3 = list(span_danger("Your throat feels very scratchy."), span_warning("Your skin feels tight."), span_danger("You can feel something move...inside."))
+	stage4 = list(span_danger("Your skin feels very tight."), span_danger("Your blood boils!"), span_danger("You can feel... something...inside you."))
```
Comment (san7890) [inline code/datums/diseases/transformation.dm:196]: Optional to multiline these two lists, but would be nice.

File: code/datums/diseases/transformation.dm [hunk 7]
```diff
@@ -314,9 +314,9 @@
 	visibility_flags = NONE
 	stage1 = list("You seem a little lighter in your step.")
 	stage2 = list("You catch yourself smiling for no reason.")
-	stage3 = list("<span class='danger'>A cruel sense of calm overcomes you.</span>", "<span class='danger'>You can't feel your arms!</span>", "<span class='danger'>You let go of the urge to hurt clowns.</span>")
-	stage4 = list("<span class='danger'>You can't feel your arms. It does not bother you anymore.</span>", "<span class='danger'>You forgive the clown for hurting you.</span>")
-	stage5 = list("<span class='danger'>You have become a Gondola.</span>")
+	stage3 = list(span_danger("A cruel sense of calm overcomes you."), span_danger("You can't feel your arms!"), span_danger("You let go of the urge to hurt clowns."))
```
Comment (san7890) [inline code/datums/diseases/transformation.dm:317]: Optional to multiline this list, but would be nice.

**Review by san7890 — CHANGES_REQUESTED** [review]
All seems good, but there's some housekeeping stuff that would be great if you could do since I don't think these files have received any attention for a few years.

File: code/datums/brain_damage/hypnosis.dm (hunk 2 above)
Comment (flowercuco) [reply code/datums/brain_damage/hypnosis.dm:35]: cant really make it fit in one line easily, i divided it in a way that at least is more up to date

**Comment by flowercuco** [comment]
i multilined all of the lists with 3 and higher and i alphabetized some stuff

File: code/datums/diseases/transformation.dm [hunk 8]
```diff
@@ -119,9 +119,13 @@
...
 	stage3 = list()
-	stage4 = list("<span class='warning'>Your back hurts.</span>", "<span class='warning'>You breathe through your mouth.</span>",
-					"<span class='warning'>You have a craving for bananas.</span>", "<span class='warning'>Your mind feels clouded.</span>")
-	stage5 = list("<span class='warning'>You feel like monkeying around.</span>")
+	stage4 = list(
+			span_warning("You breathe through your mouth."),
+			span_warning("You have a craving for bananas."),
+			span_warning("Your back hurts."),
+			span_warning("Your mind feels clouded."),
+		)
```
Comment (san7890) [inline code/datums/diseases/transformation.dm:127]:
```suggestion
	)
```

File: code/datums/diseases/transformation.dm [hunk 9]
```diff
@@ -154,10 +158,14 @@
...
-	stage2 = list("Your joints feel stiff.", "<span class='danger'>Beep...boop..</span>")
-	stage3 = list("<span class='danger'>Your joints feel very stiff.</span>", "Your skin feels loose.", "<span class='danger'>You can feel something move...inside.</span>")
-	stage4 = list("<span class='danger'>Your skin feels very loose.</span>", "<span class='danger'>You can feel... something...inside you.</span>")
-	stage5 = list("<span class='danger'>Your skin feels as if it's about to burst off!</span>")
+	stage2 = list(span_danger("Beep...boop.."), "Your joints feel stiff.")
+	stage3 = list(
+			span_danger("You can feel something move...inside."),
+			span_danger("Your joints feel very stiff."),
+			span_warning("Your skin feels loose."),
+		)
```
Comment (san7890) [inline code/datums/diseases/transformation.dm:166]:
```suggestion
	)
```

File: code/datums/diseases/transformation.dm [hunk 10]
```diff
@@ -191,10 +199,18 @@
...
-	stage2 = list("Your throat feels scratchy.", "<span class='danger'>Kill...</span>")
-	stage3 = list("<span class='danger'>Your throat feels very scratchy.</span>", "Your skin feels tight.", "<span class='danger'>You can feel something move...inside.</span>")
-	stage4 = list("<span class='danger'>Your skin feels very tight.</span>", "<span class='danger'>Your blood boils!</span>", "<span class='danger'>You can feel... something...inside you.</span>")
-	stage5 = list("<span class='danger'>Your skin feels as if it's about to burst off!</span>")
+	stage2 = list("Your throat feels scratchy.", span_danger("Kill..."))
+	stage3 = list(
+			span_danger("You can feel something move...inside."),
+			span_danger("Your throat feels very scratchy."),
+			span_warning("Your skin feels tight."),
+		)
```
Comment (san7890) [inline code/datums/diseases/transformation.dm:207]:
```suggestion
	)
```

File: code/datums/diseases/transformation.dm [hunk 11]
```diff
@@ -191,10 +199,18 @@
...
+	stage4 = list(
+			span_danger("You can feel... something...inside you."),
+			span_danger("Your blood boils!"),
+			span_danger("Your skin feels very tight."),
+		)
```
Comment (san7890) [inline code/datums/diseases/transformation.dm:212]:
```suggestion
	)
```

File: code/datums/diseases/transformation.dm [hunk 12]
```diff
@@ -314,9 +330,13 @@
...
 	stage1 = list("You seem a little lighter in your step.")
 	stage2 = list("You catch yourself smiling for no reason.")
-	stage3 = list("<span class='danger'>A cruel sense of calm overcomes you.</span>", "<span class='danger'>You can't feel your arms!</span>", "<span class='danger'>You let go of the urge to hurt clowns.</span>")
-	stage4 = list("<span class='danger'>You can't feel your arms. It does not bother you anymore.</span>", "<span class='danger'>You forgive the clown for hurting you.</span>")
-	stage5 = list("<span class='danger'>You have become a Gondola.</span>")
+	stage3 = list(
+			span_danger("A cruel sense of calm overcomes you."),
+			span_danger("You can't feel your arms!"),
+			span_danger("You let go of the urge to hurt clowns."),
+		)
```
Comment (san7890) [inline code/datums/diseases/transformation.dm:337]:
```suggestion
	)
```

File: code/datums/brain_damage/special.dm [hunk 13]
```diff
@@ -324,13 +324,14 @@
...
-												"Do you even exist?",\
-												"You simply fade away.")]</span>")
+	to_chat(owner, span_warning("[pick(list(
+			"Do you even exist?",
+			"To be or not to be...",
+			"Why exist?",
+			"You simply fade away.",
+			"You stop keeping it real.",
+			"You stop thinking for a moment. Therefore you are not.",
+		))]"))
```
Comment (san7890) [inline code/datums/brain_damage/special.dm:334]:
```suggestion
	))]"))
```

File: code/datums/brain_damage/hypnosis.dm [hunk 14]
```diff
@@ -25,14 +25,16 @@
...
-	to_chat(owner, "<span class='boldwarning'>You've been hypnotized by this sentence. You must follow these words. If it isn't a clear order, you can freely interpret how to do so,\
-										as long as you act like the words are your highest priority.</span>")
+	to_chat(owner, span_reallybig(span_hypnophrase("[hypnotic_phrase]")))
+	to_chat(owner, span_notice("[pick(list(
+			"Something about this sounds... right, for some reason. You feel like you should follow these words.",
+			"These words keep echoing in your mind. You find yourself completely fascinated by them.",
+			"You feel a part of your mind repeating this over and over. You need to follow these words.",
+			"You feel your thoughts focusing on this phrase... you can't seem to get it out of your head.",
+			"Your head hurts, but this is all you can think of. It must be vitally important.",
+		))]"))
```
Comment (san7890) [inline code/datums/brain_damage/hypnosis.dm:35]:
```suggestion
	))]"))
```

**Review by san7890 — CHANGES_REQUESTED** [review]
only pedantic stuff remains, the end parenthesis of any multi-lined list should have the same number of indents on its line as the number of indents that the `list(` started on. like this:
```txt
    var/list/my_swag_list = list( // this line has one indentation tab
        "chungus",
        "deez",
        "flarp",
    ) // so the end parenthesis should also only have one indentation tab.
```
it's not consistent in code because people probably didn't notice/care but this is definitely the better way to give it some sort of readable completeness to the whole thing

**Comment by flowercuco** [comment]
i indented all of the lists too much i think. oops

**Comment by san7890** [comment]
you missed one, i just commited it for you

**Pull request merged**